apps/remote_control/lib/lexical/remote_control/search/store/backends/khepri/operations.ex

```diff
@@ -0,0 +1,93 @@
+defmodule Lexical.RemoteControl.Search.Store.Backends.Khepri.Operations do
+  @doc """
+  Operations for working with the Khepri store
+
+  This module has a normal API, but due to the fact that khepri has to rewrite any functions
+  used in transactions, and it knows nothing about protocols, all the functions in this module
+  shy away from using any of elixir's modules that come anywhere near protocols. That means
+  No inspect, no Enum, etc. Luckily tests will fail if we include anything we shouldn't.
+  """
+  alias Lexical.RemoteControl.Search.Indexer.Entry
+  alias Lexical.RemoteControl.Search.Store.Backends.Khepri.Condition
+
+  import Condition
+
+  def tx_insert_many(entries) do
+    :lists.foreach(&:khepri_tx.put(to_path(&1), &1), entries)
```

Comment:
Enum.each

Reply:
Read the comment at the top of this file, we can't use Enum.each, because it references the Enumerable protocol.